avocado/utils/pci.py

```diff
@@ -113,6 +113,25 @@ def get_nics_in_pci_address(pci_address):
     return net_interfaces_list
 
 
+def get_pci_class_name(pci_address):
+    """
+    Gets pci class name for given pci bus address
+
+    e.g: >>> pci.get_pci_class_name("0000:01:00.0")
+             'scsi_host'
+
+    :param pci_address: Any segment of a PCI address(1f, 0000:00:if, ...)
+
+    :return: class name for corresponding pci bus address
+    """
+    pci_class_dic = {'0104': 'scsi_host', '0c04': 'fc_host',
+                     '0200': 'net', '0108': 'nvme'}
+    pci_class_id = get_pci_prop(pci_address, "Class")
+    if pci_class_id not in pci_class_dic.keys():
+        return ""
```

Comment:
Given the limited set of `pci_class_dict` I'd suggest raising an exception here. I'd recommend either `ValueError` or even `NotImplementedError` with a message `"Class ID %s is not defined in this library, please send an update" % pci_class_id`